src/pl.x86.json

```diff
@@ -217,9 +217,9 @@
 			"folder-name": "CSVLint",
 			"display-name": "CSV Lint",
 			"version": "0.4",
```

Comment:
you missed to adapt also this version to 0.4.1, see https://ci.appveyor.com/project/donho/npppluginlist-yy2vy/builds/41174235/job/rcqo1uapfmqavocv/messages